users/views.py

```diff
@@ -72,4 +72,22 @@ def post(self,request):
             return JsonResponse({"message" : "INVALID_USER"}, status = 401) 
         
         except KeyError: 
-            return JsonResponse({"message" : "KEY_ERROR"} , status = 400)
\ No newline at end of file
+            return JsonResponse({"message" : "KEY_ERROR"} , status = 400)
+
+class MyPageView(View):
```

Comment:
```suggestion
class UserView(View):
```
페이지 기준 보다는 해당 view가 어떤 데이터를 다루는지를 나타내는 것이 좋습니다.

Reply:
수정했습니다~~